ui/mods/msu/mod_settings/settings_screen.js

```diff
@@ -155,13 +155,15 @@ ModSettingsScreen.prototype.hide = function()
 
 ModSettingsScreen.prototype.show = function (_data)
 {
+	this.mIsFirstShow = true;
```

Comment:
Should be added to ModSettingsScreen() initialization, though I'm not a huge fan of this solution regardless. If you say it's necessary though w/e

Reply:
It's not great but I couldn't figure out another way to not have them show up small otherwise, it's some weird dom thing

Reply:
Fair enough, just throw it in the ModSettingsScreen inititialization then